db/queries/create_tables.sql

```diff
@@ -47,3 +48,70 @@ CREATE TABLE partidos (
   situacao VARCHAR(20) NOT NULL,
   id_bloco INTEGER REFERENCES blocos(id_bloco) NOT NULL
 )
+
+CREATE TABLE gabinete (
```

Comment:
Essa tabela parece ser inútil. Só remover a referência em deputados e tudo certo.

Reply:
Na API esses dados estavam em um objeto separado. Mas podemos fazer isso sim.